src/js/apps/patients/patient/dashboard/dashboard_app.js

```diff
@@ -58,6 +58,14 @@ export default App.extend({
   },
 
   onAddProgramFlow(programFlow) {
+    const flow = programFlow.getFlow(this.patient.id);
+
+    flow.saveAll().done(() => {
+      this.collection.unshift(flow);
```

Comment:
I don't think there's reason to add this to the collection if we're going to immediately route off the page

Reply:
was wondering about that. sounds fair